client/app/components/BottomTabs.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import { StyleSheet, View, Text, TouchableOpacity } from 'react-native';
+import { withNavigation } from 'react-navigation';
+
+import connectToRedux from '../redux/lib/connectToRedux';
+import sp from '../redux/lib/stateProperties';
+
+import styles, { colors } from '../config/styles';
+import SimpleLineIcons from '@expo/vector-icons/SimpleLineIcons';
+import MaterialCommunityIcons from '@expo/vector-icons/MaterialCommunityIcons';
+import MaterialIcons from '@expo/vector-icons/MaterialIcons';
+
+class BottomTabs extends Component {
+
+    constructor(props) {
+        super(props);
+    }
+
+    tabButtons = [
+        {
+          viewName: 'FriendsView',
+          IconComponent: SimpleLineIcons,
+          iconName: 'people',
+          opacity: this.props.viewName == 'FriendsView' ? 1 : .3,
+        },
+        {
+          viewName: 'DiningHallsView',
+          IconComponent: MaterialCommunityIcons,
+          iconName: 'food',
+          opacity: this.props.viewName == 'DiningHallsView' ? 1 : .3,
+        },
+        {
+          viewName: 'FavoritesView',
+          IconComponent: MaterialIcons,
+          iconName: 'favorite',
+          opacity: this.props.viewName == 'FavoritesView' ? 1 : .3,
+        }
+    ]
+
+    render() {
+        return (
+            <View style={tabStyles.container}>
+                <View style={{
+                    ...styles.container.backgroundColorPrimary, 
+                    ...tabStyles.bottomTabs,
+                }}>
+                    {this.tabButtons.map(tabButton => { return (
+                        <TouchableOpacity 
+                            key={tabButton.viewName} 
+                            activeOpacity={.3}
```

Comment:
Should this be `tabButton.opacity` instead of `.3`?

Reply:
^ Oooh good catch.

Reply:
activeOpacity is the opacity that we want it to change to while someone holds down the tab. It defaults to .2, but I tried out different values and liked .3 as the opacity of the tabs that you're not currently on. So I thought that the activeOpacity should match, and therefore manually set it to .3 (which if we don't like can change back to .2 but it's barely noticeable)

Reply:
The activeOpacity is just for how dark you want the icons to turn when your finger touches the light one. It's not really that important because people will mostly be tapping the already dark ones (the one you're currently on is white icon and the others are dark). The opacity in the props just determines how dark those other tabs that you're not currently on look.